src/AppInstallerCommonCore/Filesystem.cpp

```diff
@@ -152,7 +152,7 @@ namespace AppInstaller::Filesystem
 #endif
         try
         {
-            std::filesystem::create_symlink(target, link);
+            std::filesystem::create_symlink(std::filesystem::weakly_canonical(target), link);

```

Comment:
After some testing, symlinks can point to relative paths, so we should probably use whatever the target value is passed in. Caller needs to canonicalize it if caller wants to.